src/ui/components/form/FormField.tsx

```diff
@@ -33,6 +34,7 @@ export function FormField({
   className,
   help,
   id,
+  isArg,
```

Comment:
it seems you are creating this prop because you want to add some CSS to all the labels of arg inputs. Why not just add a class name here https://github.com/paritytech/contracts-ui/blob/6737d9bdbac0262f13ea9ceda6e4abbfd2f1cf33/src/ui/components/form/ArgumentForm.tsx#L24 and then use CSS to style all the labels in that container?
```
.arg-form label {
    font-mono text-xs
}
```